tests/acceptance/test_hooked_transformer.py

```diff
@@ -83,6 +83,8 @@ def test_model(name, expected_loss):
     # Runs the model on short text and checks if the loss is as expected
     model = HookedTransformer.from_pretrained(name)
     loss = model(text, return_type="loss")
+    print(4e-5)
+    print(loss.item() - expected_loss)
```

Comment:
Make sure to remove log points when you submit your PR!

Reply:
So sorry, I forgot about those. I just removed them!